docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/security-update.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Security update checker
+description: Reference information for evaluator tools.
+template: howto-guide-template
+---
+
+Security Update Checker is a tool that checks if security fixes exist for Spryker modules that are present in your project.
+
+## Problem description
+
+A project can sometimes use dependencies that contain known vulnerabilities. To minimize the security risk for the project, such dependencies should be updated to the version that has the vulnerability fixed.
```

Comment:
We need to describe that the dependencies are Spryker packages/modules and Spryker Security checker informs about such vulnerabilities very fast.